apps/web/public/static/locales/en/common.json

```diff
@@ -1465,7 +1469,7 @@
   "team_url_required": "Must enter a team URL",
   "team_url_taken": "This URL is already taken",
   "team_publish": "Publish team",
-  "number_sms_notifications": "Phone number (SMS notifications)",
+  "number_sms_notifications": "Phone number (SMS or Whatsapp notifications)",
```

Comment:
Not sure about this.

There can be SMS and WHATSAPP actions associated with the same event type so
the client may end up receiving both SMS or WHATSAPP actions (for the same event)
depending on the workflows associated with the event type. 

This means it may not just be a case of SMS or Whatsapp it could be
1. SMS
2. Whatsapp
3. SMS and WHATSAPP

Reply:
Kindly check [this](https://github.com/calcom/cal.com/blob/931763959b288776514e0b90597fe18d2dc7cf08/packages/features/bookings/lib/getBookingFields.ts#L22) and related code for context and let me know how we proceed.

Reply:
I think to keep it simple we can just call it "Text notifications" instead

Reply:
I've updated this to show "Phone Number (Text Notifications)".
Changed the related locale variable to number_text_notifications